utils/tio/Makefile

```diff
@@ -37,7 +37,7 @@ endef
 
 define Package/tio/install
 	$(INSTALL_DIR) $(1)/usr/sbin
-	$(INSTALL_BIN) $(PKG_BUILD_DIR)/src/tio $(1)/usr/sbin/
+	$(INSTALL_BIN) $(PKG_BUILD_DIR)/openwrt-build/src/tio $(1)/usr/sbin/
```

Comment:
uhhh this doesn't go in PKG_INSTALL_DIR?

Reply:
PKG_INSTALL_DIR has the same-size binary. Should I use it instead?

Reply:
Yes